### PR TITLE
Update ranch for OTP24 compatibility

### PR DESCRIPTION
**Pull request by savonarola**

### CODE REVIEW

**Review by k32 — APPROVED** [review]

**Review by qzhuyan — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

File: src/cowboy.appup.src [hunk 1]
```diff
@@ -1,8 +1,8 @@
 %% -*-: erlang -*-
 
-{"2.8.3",
+{"2.8.4",
   [
-    {"2.8.2", [
+    {<<"2\\.8\\.[23]">>, [
```
Comment (zmstone) [inline src/cowboy.appup.src]: 2-3?

**Comment by zmstone** [comment]
Ranch 1.7.1 to 1.8.0 doesn’t support appup 
So we’ll likely have to tag a 2.9.0 for cowboy so 2.8.x is preserved for emqx 4.3

File: src/cowboy.appup.src (hunk 1 above)
Comment (savonarola) [reply src/cowboy.appup.src]: Yes, in this particular case it's the same :)

**Comment by savonarola** [comment]
@zmstone so for 2.9.0 tag we just reset appup file?

**Comment by zmstone** [comment]
> @zmstone so for 2.9.0 tag we just reset appup file?

yes.

**Review by zmstone — APPROVED** [review]

**Pull request merged**